src/components/ProductList/ProductItem.1.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Image from '../../construction/Image';
+import { AddToCart, FavoriteBtn } from './../../common/Buttons';
+import global from '../../modules/global.module.scss';
+import product from '../../modules/products.module.scss';
+
+export const ProductItem = ({ image, title, price }) => {
+
+	return (
+		<div className={product.itemBody}>
+			<div className={product.boxImage}>
+				<Link to='/item_details'>
+					<div className={product.curtain}>
+						<div className={product.box}>
+							<FavoriteBtn />
```

Comment:
Use destructuring on product as well

Reply:
I don't undestand what? it is styles syntax